apps/lfx-one/src/app/modules/dashboards/executive-director/components/email-ctr-drawer/email-ctr-drawer.component.html

```
@@ -0,0 +1,89 @@
+        <div class="flex flex-col gap-1">
+          <h3 class="text-sm font-semibold text-gray-900">Monthly CTR Trend</h3>
+          <p class="text-sm text-gray-600">Email click-through rate over the last 6 months</p>
+        </div>
+        <div class="h-[240px]" data-testid="email-ctr-drawer-chart">
+          <lfx-chart type="bar" [data]="chartData()" [options]="chartOptions" height="100%"></lfx-chart>
+        </div>
+      </div>
+    }
+
+    <!-- Campaign Breakdown -->
+    @if (data().campaignGroups.length > 0) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)